### ISSUE TITLE
WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (.our-brand-prev)

### ISSUE THREAD

**Issue by jeankhoury0** [jeankhoury0/A11Y-Test-Library#2]
#### Issue
`Ensures ARIA attributes are allowed for an element's role` ([aria-allowed-attr](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr))
#### Target application
[Oral Health and Dental Care | Colgate®](https://www.colgate.com/en-us)
#### Element path
.our-brand-prev
#### Snippet
`<div class="our-brand-prev slick-arrow slick-disabled" aria-label="Previous Arrow" aria-disabled="true" style="display: block;"></div>`
#### How to fix
    Fix all of the following:
      ARIA attribute cannot be used, add a role attribute or use a different element: aria-label
#### Environment
Chrome version 91.0.4472.164
This accessibility issue was found using Accessibility Insights for Web 2.27.0 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
